CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to `Jarvis-personal-assistant`. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by this [Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [jarvis.passistant@gmail.com](mailto:jarvis.passistant@gmail.com).
+
+## How Can I Contribute?
+
+### Reporting Bugs
+
+This section guides you through submitting a bug report. Following these guidelines helps maintainers and the community understand your report :pencil:, reproduce the behavior :computer: :computer:, and find related reports :mag_right:.
+
+When you are creating a bug report, please include as many details as possible. Fill out [the required template](https://github.com/Harkishen-Singh/Jarvis-personal-assistant/blob/master/.github/ISSUE_TEMPLATE/bug_report.md), the information it asks for helps us resolve issues faster.
+
+> **Note:** If you find a **Closed** issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
+
+### Suggesting Enhancements
+
+This section guides you through submitting an enhancement suggestion, including completely new features and minor improvements to existing functionality. Following these guidelines helps maintainers and the community understand your suggestion :pencil: and find related suggestions :mag_right:.
+
+When you are creating an enhancement suggestion, please include as many details as possible. Fill in [the template](https://github.com/Harkishen-Singh/Jarvis-personal-assistant/blob/master/.github/ISSUE_TEMPLATE/feature_request.md), including the steps that you imagine you would take if the feature you're requesting existed.
+
+### Your First Code Contribution
+
+Unsure where to begin contributing? You can start by looking through these `beginner` and `help-wanted` issues:
+
+* Beginner issues - issues which should only require a few lines of code, and a test or two.
+* Help wanted issues - issues which should be a bit more involved than `beginner` issues.
+
+### Pull Request Process
+1. Make a new branch for each feature/pull request. 
+2. Ensure any install or build dependencies are removed before the end of the layer when doing a build.
+3. Ensure that the code follows the linting standards. `eslint` is used for the JavaScript files and `golint` is used for the Golang files.
+4. Write appropriate comments in the pull request, explaining the changes made. **Reference the issue you are fixing**.
+5. For changes to the front-end include screenshots or screencasts showing the working changes. You can use browser extensiosns like `Screencastify` to create screencasts.
```

Comment:
Can you write in the line as `would be really helpful for reviewing` or anything similar?

Reply:
@Harkishen-Singh Done.